source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -13,17 +13,215 @@ Document Data Format: POJOs
 Overview
 --------
 
-- Description; convert between BSON and POJOs  
-- Summary of contents of this doc
+This guide explains how you can store and retrieve MongoDB data using plain
+old Java objects (`POJOs <https://en.wikipedia.org/wiki/Plain_old_Java_object>`__).
+POJOs are classes that do not depend on any other files or frameworks, and are
```

Comment:
Nitpick: "any other" seems vague to me -- does this mean that I can't extend a class in my project to create a POJO if it lives in another file? I would suggest "third party" or "external" as potential alternatives.

Reply:
I guess I wasn't successful in trying to provide a concise description of POJOs. Thanks for the suggestions, I'll try to rework the description.